networking/configuring-ptp.adoc

```diff
@@ -5,13 +5,18 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-:FeatureName: Precision Time Protocol (PTP) hardware
+:FeatureName: PTP single NIC hardware with boundary clock
 include::modules/technology-preview.adoc[leveloffset=+1]
 
+[TIP]
```

Comment:
Looking at the SSG's guidance on admonition types, I think this might be a note and not a tip. 
https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

Also, we have three admonitions right at the top of the page. For note admonitions close together, ISG recommends combining them into one admonition as a bulleted list. https://www.ibm.com/docs/en/ibm-style?topic=format-notes#standard-notes

Unfortunately, you need to keep the title of the call out as `NOTE` because `NOTES` can cause AsciiDoc problems.